.gitignore

```diff
@@ -1 +1,2 @@
-venv/
\ No newline at end of file
+venv/
+**__pycache__/
```

Comment:
Is this the right way to ignore `pycache` folder ?. It should be `__pycache__/`

Reply:
The pycache folder is appearing in the tests folder after I run tests so I think if I wanted to make it more specific I would write `tests/__pycache__/`.  The ** one ignores the pycache folder in all directories. Which one do you think is better?

Reply:
@ethanschreur use the suggestion I have given. It is better.